automatic/apache-httpd/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,6 @@
+$configFile = Join-Path $env:chocolateyPackageFolder 'config.json'
+$config = Get-Content $configFile -Raw | Out-String | ConvertFrom-Json
```

Comment:
You can't use ConvertFrom-Json, scripts are meant to be PSv2 compatible and ConvertFrom-Json was added in PSv3